contracts/tests/test_locker.cairo

```diff
@@ -0,0 +1,119 @@
+use unruggablememecoin::locker::{ILockerDispatcher, ILockerDispatcherTrait};
+use unruggablememecoin::sample_erc20::{SampleERC20, IERC20Dispatcher, IERC20DispatcherTrait};
+use result::ResultTrait;
+use snforge_std::{ declare, ContractClassTrait, start_prank, stop_prank, PrintTrait, start_warp, stop_warp, CheatTarget};
+use array::ArrayTrait;
+use starknet::ContractAddress;
+
+struct Setup {
+    locker: ILockerDispatcher,
+    lp_token: IERC20Dispatcher,
+    owner: ContractAddress
+}
+
+fn setup() -> Setup {
+    let owner_address: felt252 =
+        0x123123123123123123123123123;
+    let locker_contract = declare('Locker');
+
+    let mut locker_calldata = ArrayTrait::new();
+    locker_calldata.append(200_u64.into()); // 200ms lock
```

Comment:
use mock timestamp to not rely on actual time passed

Reply:
Isnt it mock already?